exporters/common/src/main/java/io/opentelemetry/exporter/internal/marshal/Marshaler.java

```diff
@@ -38,6 +38,14 @@ public final void writeJsonTo(JsonGenerator output) throws IOException {
     }
   }
 
+  /** Marshals into the {@link JsonGenerator} in proto JSON format and adds a newline. */
+  public final void writeJsonWithNewline(JsonGenerator output) throws IOException {
```

Comment:
Should this code call `output.writeRaw('\n')` or should `StreamJsonWriter`? Seems like an implementation detail of `StreamJsonWriter` to me.

Reply:
I tried calling from StreamJsonWriter, but this method closes the generator, so it fails

Reply:
@jack-berg can you check again?